src/main/java/school/lemon/changerequest/java/Fraction_Number/FractionNumber.java

```diff
@@ -0,0 +1,139 @@
+package school.lemon.changerequest.java.Fraction_Number;
+
+/**
+ * Created by User on 07.12.2016.
+ */
+public class FractionNumber {
+    public FractionNumber() {
+    }
+
+    public FractionNumber(int Numerator, int Denominator) {
```

Comment:
Please read java convention. All variables, fields and method arguments should be named on `lowerCamelCase`.